src/components/Engines/Engines.test.tsx

```diff
@@ -63,9 +63,8 @@ describe('<Engines /> component', () => {
       },
     };
 
-    mockPackageMeta.mockImplementation(() => packageMeta);
-
-    const wrapper = mount(<Engine />);
-    expect(wrapper.html()).toEqual('');
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const wrapper = mount(withEngineComponent(packageMeta as any));
```

Comment:
same as above why `any`?

Reply:
I was aiming for minimal changes to the test fixtures (which were not typed, hence this `any`).

I've updated the fixtures now to make it not necessary.